### PR TITLE
County mapping to Tracts

### LINKED ISSUE #874: Convert county mapping to use tracts, not CBGs
We should look into whether we can use Geocorr (which we use for other mapping earlier in the process). Maybe we  don't want to do that though in case we drop Geocorr in the future. Whoever takes this should figure this out by leading the covnersation and understanding what we get from Geocorr and when.

### PR DESCRIPTION
**Pull request by esfoobar-usds**
Fixes #874

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/score/etl_score_post.py [hunk 1]
```diff
@@ -58,12 +60,14 @@ def _extract_score(self, score_path: Path) -> pd.DataFrame:
...
+    def _extract_national_tract(
+        self, national_tract_path: Path
+    ) -> pd.DataFrame:
+        logger.info("Reading national tract file")
         return pd.read_csv(
-            national_cbg_path,
-            names=["GEOID10"],
-            dtype={"GEOID10": "string"},
+            national_tract_path,
+            names=[self.GEOID_TRACT_FIELD_NAME],
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/score/etl_score_post.py:69]: Love seeing all these constants change, so clean and clear.

File: data/data-pipeline/data_pipeline/etl/score/etl_score_post.py [hunk 2]
```diff
@@ -150,15 +155,19 @@ def _create_score_data(
...
         # merge state + county with score
         score_county_state_merged = score_df.merge(
-            county_state_merged, on="GEOID", how="left"
+            county_state_merged,
+            on="GEOID",  # GEOID is the county ID
+            how="left",
         )
 
         # check if there are census cbgs without score
         logger.info("Removing CBG rows without score")
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/score/etl_score_post.py:164]: nit: should we switch all references to CBGs to tracts?

**Review by lucasmbrown-usds — APPROVED** [review]
TBH, I don't fully understand each aspect of the work in these parts of the pipelines, but the code LGTM.

File: data/data-pipeline/data_pipeline/etl/score/etl_score_post.py (hunk 2 above)
Comment (esfoobar-usds) [reply data/data-pipeline/data_pipeline/etl/score/etl_score_post.py:164]: Yes inded

**Pull request merged**